Something/WebApplication/wwwroot/js/apps/onboarding/OnboardingEntry.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import { Onboarding } from "./Onboarding";
+
+ReactDOM.render(
+    <Onboarding />,
+    document.getElementById("onboarding-root"),
```

Comment:
You can pass the props here instead of giving the responsibility to the Onboarding component.